data.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Holds the data associated with a website. Websites are identified by origin.
```

Comment:
What is "origin"? Can you add a bit of a description?

Reply:
Added a couple reference, examples, and a bit more justification.